vlib/mysql/README.md

```diff
@@ -0,0 +1,30 @@
+For Linux, you need to install `MySQL development` package and `pkg-config`.
+For Windows, install [the installer](https://dev.mysql.com/downloads/installer/) , then copy the `include` and `lib` folders to `thirdparty/mysql` on V root folder.
```

Comment:
"folders to `thirdparty/mysql` on V root folder." -> "folders to the `<V install directory>\thirdparty\mysql` folder.